go-controller/pkg/metrics/master.go

```diff
@@ -227,31 +230,50 @@ func RegisterMasterMetrics() {
 	})
 }
 
-func scrapeOvnTimestamp() float64 {
-	output, stderr, err := util.RunOVNSbctl("--if-exists",
-		"get", "SB_Global", ".", "options:e2e_timestamp")
-	if err != nil {
-		klog.Errorf("Failed to scrape timestamp: %s (%v)", stderr, err)
+func scrapeOvnTimestamp(ovnSBClient goovn.Client) func() float64 {
+	return func() float64 {
```

Comment:
@abhat instead of a function that returns a function, I'd almost rather this just go directly into the caller...

Reply:
+1 its only used 1 time anyway

Reply:
If it's better style, sure. I thought this was better for readability. the metrics registration takes a func that takes no arguments. and we need ovnSBClient to be passed in.